src/common/helper.js

```diff
@@ -554,6 +554,33 @@ function * postToBusApi (payload) {
   yield busApiClient.postEvent(payload)
 }
 
+/**
+ * Function to remove metadata details from reviews for members who shouldn't see them
+ * @param  {Array} reviews The reviews to remove metadata from
+ * @param  {Object} authUser The authenticated user details
+ */
+function cleanseReviews (reviews, authUser) {
+  const cleansedReviews = []
+
+  if (!authUser.scopes) {
+    _.forEach(reviews, (review) => {
+      const admin = _.filter(authUser.roles, role => role.toLowerCase() === 'Administrator'.toLowerCase())
```

Comment:
why we are checking the role for each review? the role would remain the same for the entire process.
hence, we should first check the role and then filter out the review

Reply:
Makes sense.